x-pack/plugins/search_indices/public/components/index_documents/add_documents_code_example.tsx

```diff
@@ -95,33 +95,27 @@ export const AddDocumentsCodeExample = ({
               onSelectLanguage={onSelectLanguage}
             />
           </EuiFlexItem>
-          {selectedLanguage === 'curl' && (
```

Comment:
I did this change in my documents [PR](https://github.com/elastic/kibana/pull/196772) since it was a part of the AC for that issue.

Reply:
thanks sorry should of checked. will remove